bridgestyle/qgis/togeostyler.py

```diff
@@ -428,20 +428,25 @@ def _basePointSimbolizer(sl, opacity):
 
     return symbolizer
 
-wknReplacements = {"regular_star":"star",
+wknReplacements = {"regularstar":"star",
                "cross2": "x",
                "equilateral_triangle": "triangle",
                "rectangle": "square",
-               "filled_arrowhead": "ttf://Webdings#0x34",
+               "arrowhead": "shape://oarrow",
+               "filledarrowhead": "shape://coarrow",
                "line": "shape://vertline",
-               "arrow": "ttf://Wingdings#0xE9",
-               "diamond": "ttf://Wingdings#0x75",
-                "horline":"shape://horline",
-               "vertline":"shape://vertline",
-               "cross":"shape://plus",
-               "slash":"shape://slash",
-               "backslash":"shape://backslash",
-               "x": "shape://times"}
+               "plus":"cross",
+               # somehow these marks require the shape:// prefix in geoserver
```

Comment:
It is because they are not part of the SLD specification, see [Marks](https://docs.geoserver.org/stable/en/user/styling/sld/extensions/pointsymbols.html#marks)

Reply:
maybe, but notice the qgis marks, they have a qgis:// prefix but they can be called without a prefix